src/test/java/org/cryptomator/integrations/uiappearance/WinUiAppearanceProviderTest.java

```diff
@@ -0,0 +1,47 @@
+package org.cryptomator.integrations.uiappearance;
+
+import org.cryptomator.windows.uiappearance.WinUiAppearanceProvider;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+@EnabledOnOs(OS.WINDOWS)
+@DisabledIfEnvironmentVariable(named = "CI", matches = ".*")
+public class WinUiAppearanceProviderTest {
+
+	private WinUiAppearanceProvider appearanceProvider;
+
+	@BeforeEach
+	public void setup() {
+		this.appearanceProvider = new WinUiAppearanceProvider();
+	}
+
+	@Test
+	@DisplayName("get current system theme")
+	public void testGetSystemTheme() {
+		Theme myTheme = appearanceProvider.getSystemTheme();
+
+		Assertions.assertNotNull(myTheme);
+		System.out.println("current theme: " + myTheme.name());
+	}
+
+	@Test
+	@DisplayName("add theme listener, wait 10s, remove theme listener")
+	@Disabled
```

Comment:
Since the whole class is already disabled for CI, maybe this test can be enabled again? I know that it is long running, but locally this does not hurt imho.

Reply:
This test only makes sense when running it manually (since one would want to change the system theme). But it might still make sense to run other tests in this class locally (during `mvn test`).